arches/app/models/tile.py

```diff
@@ -525,7 +530,10 @@ def delete(self, *args, **kwargs):
             try:
                 super(Tile, self).delete(*args, **kwargs)
                 for nodeid in self.data.keys():
-                    node = models.Node.objects.get(nodeid=nodeid)
+                    try:
+                        node = SimpleNamespace(**next((x for x in self.serialized_graph["nodes"] if x["nodeid"] == nodeid), None))
+                    except:
```

Comment:
It looks like this is probably a TypeError we are excepting here if self.serialized_graph is None. If that's the case, I think we should probably be explicit about that even if the odds of other errors are very low.